texar/torch/modules/connectors/connectors.py

```diff
@@ -302,8 +302,7 @@ class ForwardConnector(ConnectorBase):
     :attr:`output_size`, or must have the same number of elements and be
     re-packable into the structure of :attr:`output_size`. Note that if input
     is or contains a ``dict`` instance, the keys will be sorted to pack in
-    deterministic order (See :func:`~texar.torch.utils.nest.pack_sequence_as`
-    for more details).
+    deterministic order.
```

Comment:
"pack in deterministic order" is a bit too vague. We can either make `pack_sequence_as` public in docs and refer to it, or copy docs of `pack_sequence_as` here to make it clear. (The former is probably preferred).